assets/js/blocks/featured-product/index.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+import { registerBlockType } from '@wordpress/blocks';
+import { getSetting } from '@woocommerce/settings';
+import { isFeaturePluginBuild } from '@woocommerce/block-settings';
+import { Icon, starEmpty } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { example } from './example';
+import { Edit } from './edit';
+import metadata from './block.json';
+
+/**
+ * Register and run the "Featured Product" block.
+ */
+registerBlockType( metadata, {
+	icon: {
+		src: (
+			<Icon
+				icon={ starEmpty }
+				className="wc-block-editor-components-block-icon"
+			/>
+		),
+	},
+	attributes: {
+		...metadata.attributes,
+		/**
+		 * A minimum height for the block.
+		 *
+		 * Note: if padding is increased, this way the inner content will never
+		 * overflow, but instead will resize the container.
+		 *
+		 * It was decided to change this to make this block more in line with
+		 * the “Cover” block.
+		 */
+		minHeight: {
+			type: 'number',
+			default: getSetting( 'default_height', 500 ),
+		},
+	},
+	supports: {
+		...metadata.supports,
+		color: {
+			background: false,
+			text: true,
```

Comment:
Technically, here and below on L57, you can spread it again. It doesn't save many lines of code but it makes the intent more clear (i.e. we are only overriding __experimental stuff) and it makes sure any update on the .json is retained here.

EDIT: Oh, I see why now. Because you had to add the __experimental stuff also in the block.json, it would be spread here as well. Since the block is registered here, wouldn't it be enough to add them at runtime here and remove them from block.json?

Reply:
@sunyatasattva I thought that too, but if it's not added on the `block.json` too, the setting (duotone, in this case) is not applied when rendering the block on the front-end.

Reply:
Right. It makes sense.